tests/TestHelper.php

```diff
@@ -95,6 +102,41 @@ function ($method, $absUrl, $headers, $params, $hasFile) {
         ;
     }
 
+    /**
+     * Sets up a request expectation with the provided parameters. The request
+     * will actually go through and be emitted.
+     *
+     * @param string $method HTTP method (e.g. 'post', 'get', etc.)
+     * @param string $path relative path (e.g. '/v1/charges')
+     * @param null|array $params array of parameters. If null, parameters will
+     *   not be checked.
+     * @param null|string[] $headers array of headers. Does not need to be
+     *   exhaustive. If null, headers are not checked.
+     * @param bool $hasFile Whether the request parameters contains a file.
+     *   Defaults to false.
+     * @param null|string $base base URL (e.g. 'https://api.stripe.com')
+     */
+    protected function expectsRequestStream(
```

Comment:
Should we add a test that uses this? Maybe on Quotes?

Reply:
Yes, I will rebase [`latest-codegen`](https://github.com/stripe/stripe-php/pull/1152/commits/b4ed515e9d52a124ae02afc5ec6baf0d19d9c0fa) after this is merged, and that PR contains such a test.

Wanted to separate out the infra stuff in this PR vs. the actual quotes-related functionality.